sdk/playwrighttesting/microsoft-playwright-testing/src/common/constants.ts

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+export const EntraIdAccessTokenConstants = {
+  LIFETIME_LEFT_THRESHOLD_IN_MINUTES_FOR_ROTATION: 15,
+  SCOPE: "https://management.core.windows.net/.default",
+  ROTATION_INTERVAL_PERIOD_IN_MINUTES: 4,
+};
+
+/** @public
+ *
+ * OS types supported on Microsoft Playwright Testing cloud hosted browsers
+ */
+export const ServiceOsConstants = {
+  LINUX: "linux",
+  WINDOWS: "windows",
+} as const;
+
+/** @public
+ *
+ * Authentication types supported on Microsoft Playwright Testing
+ */
+export const ServiceAuthenticationConstants = {
+  ENTRA_ID: "ENTRA_ID",
+  SERVICE_TOKEN: "SERVICE_TOKEN",
+} as const;
+
+/** @public
+ *
+ * Environment variables used by Microsoft Playwright Testing
+ */
+export const ServiceEnvironmentVariableConstants = {
+  PLAYWRIGHT_SERVICE_OS: "PLAYWRIGHT_SERVICE_OS",
+  PLAYWRIGHT_SERVICE_RUN_ID: "PLAYWRIGHT_SERVICE_RUN_ID",
+  PLAYWRIGHT_SERVICE_EXPOSE_NETWORK_ENVIRONMENT_VARIABLE: "PLAYWRIGHT_SERVICE_EXPOSE_NETWORK",
+  PLAYWRIGHT_SERVICE_ACCESS_TOKEN: "PLAYWRIGHT_SERVICE_ACCESS_TOKEN",
+  PLAYWRIGHT_SERVICE_URL: "PLAYWRIGHT_SERVICE_URL",
+  PLAYWRIGHT_SERVICE_REPORTING_URL: "PLAYWRIGHT_SERVICE_REPORTING_URL",
+};
+
+export const DefaultConnectOptionsConstants = {
+  DEFAULT_TIMEOUT: 0,
+  DEFAULT_SLOW_MO: 0,
+  DEFAULT_EXPOSE_NETWORK: "<loopback>",
+  DEFAULT_SERVICE_OS: ServiceOsConstants.LINUX,
+};
+
+export const API_VERSION = "2023-10-01-preview";
+
+// Do not put an ending slash for the urls,
+// for example use https://www.microsoft.com/en-in and not https://www.microsoft.com/en-in/
+
+export class Constants {
+  // Config related constants
+  public static readonly TEST_FRAMEWORK_NAME = "Playwright";
+  public static readonly TEST_FRAMEWORK_RUNNERNAME = "Playwright Test Runner";
+  public static readonly TEST_TYPE = "WebTest";
+  public static readonly TEST_SDK_LANGUAGE = "JavaScript/TypeScript";
+  // Placeholder version
+  public static readonly REPORTER_PACKAGE_VERSION = "0.0.4";
+  public static readonly DEFAULT_DASHBOARD_ENDPOINT = "https://playwright.microsoft.com";
+  public static readonly DEFAULT_SERVICE_ENDPOINT =
+    "https://{region}.reporting.api.playwright-test.io";
```

Comment:
if fully shifted change it to https://{region}.reporting.api.playwright.microsoft.com

Reply:
this is the test domain.